cmds/dhclient/dhclient.go

```diff
@@ -37,6 +37,7 @@ const (
 
 var (
 	leasetimeout = flag.Int("timeout", 15, "Lease timeout in seconds")
+	retry        = flag.Int("retry", -1, "Max number of attempts to send DHCPv6 solicit. -1 means infinity")
```

Comment:
That's good, though Ron does not care about DHCPv4 anymore I suppose. But I will put your idea on DHCPv4 client still.